ansys/mapdl/core/launcher.py

```diff
@@ -585,10 +584,6 @@ def save_ansys_path(exe_loc=''):
     # otherwise, query user for the location
     with open(CONFIG_FILE, 'w') as f:
         print('Cached ANSYS executable not found')
```

Comment:
We still need this here, just not inside the try/except block since 
```suggestion
        print('Cached ANSYS executable not found')
        exe_loc = input('Enter location of ANSYS executable: ')
```